### DISCUSSION TITLE
interactions and commands

### DISCUSSION THREAD

**Discussion by Ruttie2006** [serenity-rs/serenity#2368] [Q&A] [answered]
I'm new to rust so I don't even know if this is possible, but:

I come from C# and Discord.Net where we can create an InteractionService and have a module where we can just create commands as methods, is there something similar for this discord library?
[link](https://discordnet.dev/guides/int_framework/intro.html)

**Comment by kangalio** [comment, marked as answer]
See https://github.com/serenity-rs/poise/
